hazelcast/src/main/java/com/hazelcast/map/impl/MapKeyLoader.java

```diff
@@ -320,6 +334,10 @@ public void setHasBackup(boolean hasBackup) {
         this.hasBackup = hasBackup;
     }
 
+    public void setOperationProvider(MapOperationProvider operationProvider) {
```

Comment:
It is better to inject this through the constructor. Please fix that later if possible.